superset-frontend/packages/superset-ui-core/src/query/types/QueryResponse.ts

```diff
@@ -83,6 +83,7 @@ export interface ChartDataResponseResult {
 export interface TimeseriesChartDataResponseResult
   extends ChartDataResponseResult {
   data: TimeseriesDataRecord[];
+  label_map: Record<string, DataRecordValue[]>;
```

Comment:
nits:
```suggestion
  label_map: Record<string, string[]>;
```

Reply:
I'm using `DataRecordValue` to keep compatibility with `labelMap` defined in `EChartTransformedProps`.

Reply:
the value of `label_map` represents parts of columns, it will not be null or Date.

Reply:
Got it. I'll open a follow-up to fix all `labelMap` references then.